debug/src/lib.rs

```diff
@@ -65,6 +60,19 @@ impl From<DebugColor> for Color {
     fn from(DebugColor(color): DebugColor) -> Self {
         color
     }
+} */
```

Comment:
You can remove the code rather than commenting it ;-)

Reply:
Yeah I was keeping it around to check how things were before but that's pretty unnecessary.